test/OpenTelemetry.Api.Tests/OpenTelemetry.Api.Tests.csproj

```diff
@@ -27,5 +27,7 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Coyote" />
+    <!-- System.Text.Json is indirect reference. It is needed to upgrade it directly to avoid https://github.com/advisories/GHSA-hh2w-p6rv-4g7w -->
```

Comment:
Consider creating a tracking issue, so we can remove these once Microsoft.Coyote made the fix/update.

Reply:
#5815